assets/eip-4675/contracts/ERC20Token.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: CC0-1.0
```

Comment:
i think we shall ask the author to grant it with new license....

Reply:
We did ask, and the author approved.